components/gitpod-protocol/src/experiments/configcat-server.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Client } from "./types";
+import * as configcat from "configcat-node";
+import { LogLevel } from "configcat-common";
+import { ConfigCatClient } from "./configcat";
+import { newAlwaysReturningDefaultValueClient } from "./always-default";
+
+let client: Client | undefined;
+
+export function getExperimentsClientForBackend(): Client {
+    // We have already instantiated a client, we can just re-use it.
+    if (client !== undefined) {
+        return client;
+    }
+
+    // Retrieve SDK key from ENV Variable
+    const sdkKey = process.env.CONFIGCAT_SDK_KEY;
+
+    // Self-hosted installations do not set the ConfigCat SDK key, so always use a client which returns the default value.
+    if (sdkKey == undefined || sdkKey === "") {
```

Comment:
nit: `==` always feels like a bug, and IMO we should never use it. `===` is equivalent in this case anyway.

Reply:
Thanks, this was a miss on my part.

Reply:
Fixed.